tensorboard/plugins/mesh/summary.py

```diff
@@ -157,19 +175,23 @@ def pb(name,
   json_config = _get_json_config(config_dict)
 
   summaries = []
-  tensors = [(vertices, plugin_data_pb2.MeshPluginData.VERTEX, tf.float32),
-             (faces, plugin_data_pb2.MeshPluginData.FACE, tf.int32),
-             (colors, plugin_data_pb2.MeshPluginData.COLOR, tf.uint8)]
-  for tensor, content_type, data_type in tensors:
+  tensors = [(vertices, plugin_data_pb2.MeshPluginData.VERTEX),
```

Comment:
Since this is used in various places now, I think using a namedtuple would make this more readable/manageable.

Reply:
Good point